Bugfix for test issues

cklb opened the pull request:
This PR fixes #24 and #21

cklb commented:
@riemarc cleaned up and ready from my point of view

riemarc commented:
The test `StringMassTest.test_fem` is tested twice on each build? To prevent this and to prevent global variables you could handle `example_data` as class attribute from `StringMassTest` and skip `test_fem` with `@unittest.skipIf(StringMassTest.example_data is not None)` or something like that. Just merge this pr when you are happy with it.

cklb commented:
I changed to setup and added a class attribute to the visualization test. You're right `test_fem` is now called twice, but this way the coupling between the different tests stays low.
